Week1/work.py

```diff
@@ -37,7 +45,10 @@ def sqrt_of_numbers(num):
     '''
 
     ## Code Here
-    return None
+     if num<0:
```

Comment:
because you have copied your code from your idle its obvious there is a leading space in the start of all the code. This inconsistency is not tolerated in python so make the changes with respect to that